modules/conversion-center/module.php

```diff
@@ -110,8 +108,33 @@ public function __construct() {
 			return $cpts;
 		} );
 
+		add_action( 'admin_init', function () {
+			$action = filter_input( INPUT_GET, 'action' );
+
+			if ( 'set_as_homepage' === $action ) {
+				$post = filter_input( INPUT_GET, 'post', FILTER_VALIDATE_INT );
+				check_admin_referer( 'set_as_homepage_' . $post );
+				update_option( 'page_on_front', $post );
+				update_option( 'show_on_front', 'page' );
+				$menu_args = $this->get_menu_args();
+
+				wp_redirect( $menu_args['menu_slug'] );
+				exit;
```

Comment:
```suggestion
				wp_die();
```

Reply:
@louiswol94 this change breaks the code, it needs to be `exit `or `die()`. Reverted it.

Reply:
@nicoladj77 Is this part still needed now that the Dropdown is removed?